src/components/common/Slider/Slider.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import Slider from './Slider'
+
+jest.mock('@material-ui/core/Slider', () => (props: any) => {
+  const { startValue, min, max, step, onChange } = props
+  return (
+    <input
+      type="range"
+      defaultValue={startValue}
+      min={min}
+      max={max}
+      step={step}
+      onChange={event => onChange(event.target.value)}
+    />
+  )
+})
```

Comment:
Why do you mock the material-ui slider?

Reply:
Because materialUI replaces the input by spans and fakes it to be a slider. In order to test the functionality of an input that acts like a slider with onChange I mocked it to be actually one, since I don't want to test the 3rd Party comp itself but rather the function that I pass